sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -1445,6 +1462,204 @@ public async Task UpdateSqlServerDataFeedWithCredentialAuthentication(string aut
             Assert.That(updatedDataSource.DataSourceCredentialId, Is.EqualTo(credentialId));
         }
 
+        [RecordedTest]
+        [TestCaseSource(nameof(DataFeedSourceTestCases))]
+        public async Task UpdateCommonMembersWithNullSetsToDefault(string dataSourceKind)
+        {
+            // https://github.com/Azure/azure-sdk-for-net/issues/21623
+            if (dataSourceKind == nameof(DataFeedSourceKind.AzureEventHubs))
+            {
+                Assert.Ignore();
+            }
+
+            MetricsAdvisorAdministrationClient adminClient = GetMetricsAdvisorAdministrationClient();
+
+            string dataFeedName = Recording.GenerateAlphaNumericId("dataFeed");
+            DataFeedSource dataSource = CreateMockDataFeedSource(dataSourceKind);
+            var dataFeedToCreate = new DataFeed()
```

Comment:
Set initial values for required properties and for anything that can be updated.